src/providers/test-data-by-category/test-data-by-category.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { CategoryCode } from '@dvsa/mes-test-schema/categories/common';
+import { TestCategory } from '@dvsa/mes-test-schema/category-definitions/common/test-category';
+import { getTestData as getTestDataC } from '../../modules/tests/test-data/cat-c/test-data.cat-c.reducer';
+import { getTestData as getTestDataC1 } from '../../modules/tests/test-data/cat-c/test-data.cat-c1.reducer';
+import { getTestData as getTestDataCE } from '../../modules/tests/test-data/cat-c/test-data.cat-ce.reducer';
+import { getTestData as getTestDataC1E } from '../../modules/tests/test-data/cat-c/test-data.cat-c1e.reducer';
+
+@Injectable()
+export class TestDataByCategoryProvider {
+  public getTestDataByCategoryCode(category: CategoryCode) {
+    switch (category) {
+      case TestCategory.C: return getTestDataC;
+      case TestCategory.C1: return getTestDataC1;
+      case TestCategory.CE: return getTestDataCE;
+      case TestCategory.C1E: return getTestDataC1E;
+      default: return getTestDataC;
```

Comment:
Does this default is a safe thing to do here? 
Are we planning to reuse this provider in other categories (eg. D?) in the future? If yes, defaulting to C might be a risky thing ... even for one of the subcategories of C it seems a bit off imho...

Reply:
Yes, this will be extended for CAT D too. I'm happy to take the default out.